Assets/LeapMotion/Scripts/LeapServiceProvider.cs

```diff
@@ -177,16 +203,23 @@ public class LeapServiceProvider : LeapProvider {
       }
 #endif
 
+      if (_updateHandInPrecull!=_prevUpdateHandInPrecull && !_updateHandInPrecull) {
+        _transformArray[0] = Matrix4x4.identity;
+        _transformArray[1] = Matrix4x4.identity;
+        Shader.SetGlobalMatrixArray("_handTransforms", _transformArray);
+      }
+      _prevUpdateHandInPrecull = _updateHandInPrecull;
+
       _fixedOffset.Update(Time.time - Time.fixedTime, Time.deltaTime);
 
       if (_useInterpolation) {
 #if UNITY_ANDROID
-        Int64 time = leap_controller_.Now() - (_interpolationDelay+16) * 1000;
+        Int64 time = leap_controller_.Now() - ((_interpolationDelay+16) * 1000) + (_updateHandInPrecull ? (long)(Time.smoothDeltaTime * S_TO_NS / Time.timeScale) : 0);
```

Comment:
This will break if timescale is set to 0.  Might need to calculate your own smoothed delta based on real time.